pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -387,6 +387,10 @@ func BuildHTTPRoutesForVirtualService(
 			}
 		}
 		if catchall {
+			// log the skipped remaining routes in vs.Http to avoid silent skip
+			if i < len(vs.Http) {
+				log.Warnf("Skipping %d routes for %s due to the catch all route", len(vs.Http)-i, virtualService.Name)
+			}
```

Comment:
@howardjohn This looks like the missing log I was looking for when debuging https://github.com/istio/istio/pull/50109

Reply:
I think more folks will run into this issue after upgrade istio to 1.17 or above and this log is quite helpful for troubleshooting. @howardjohn  any suggestion to move forward?